tests/subsys/dsp/utils/prj.conf

```
@@ -0,0 +1,10 @@
+CONFIG_ZTEST=y
+CONFIG_DSP=y
+CONFIG_DSP_BACKEND_CMSIS=y
+CONFIG_CMSIS_DSP=y
+CONFIG_CMSIS_DSP_SUPPORT=y
+CONFIG_REQUIRES_FULL_LIBC=y
+
+#To print floating point numbers, the full libc is required
+CONFIG_PICOLIBC=y
+CONFIG_REQUIRES_FLOAT_PRINTF=y
```

The comment is misplaced, but you only really need the below, AFAICT (just tested it, too)

```suggestion
CONFIG_REQUIRES_FLOAT_PRINTF=y
```

---

Nice catch! Thanks